bin/restart

```diff
@@ -1,14 +1,19 @@
-#!/bin/sh
+#!/bin/bash
+
+if [[ ! -e 'docker-compose.yml' ]]; then
+  echo "Please make sure you are in a specific chapter directory with a docker compose file."
+  exit 1
+fi
 
 docker compose down
 
 # kill all running containers
-docker ps | grep `basename $(pwd)` | awk '{ print $1 }' |  xargs docker container kill
+docker ps | grep "build-your-own-internet" | awk '{ print $1 }' |  xargs docker container kill | xargs docker container rm
```

Comment:
I added this `docker container rm` because there's a lingering container once you kill it. If you jump between chapters with the same container names, you'll get a `container already exists` error message. this just tidies up the work. 

I doubled down on fixing this problem by also giving the containers/images/networks chapter specific names, e.g. `build-your-own-internet-001-boudi`. They still match the regex, but they have unique names, just in case that could bite us in the butt again later.